lib/controllers/src/Velocity.cpp

```diff
@@ -0,0 +1,14 @@
+#include "Velocity.h"
+
+using namespace Controller;
+
+bool Velocity::update() {
+  if (shouldUpdate()) {
```

Comment:
I'm a bit confused about this shouldUpdate function. if we shouldn't update why is the function to update being called in the first place. Should the shouldUpdate function be called outside the function before calling this one?

Reply:
you're right that is confusing. ig my hope was that the API was clean. the use of ActuatorController derived classes should just call update() and that function is smart enough to figure out if it should update. i think having if(shouldUpdate()) update() everywhere might make the code less readable at the main.cpp lvl